region/lowernorfair/east.json

```diff
@@ -125,9 +141,34 @@
           "nodeSubType": "blue",
           "sparking": {
             "runways": [
+              {
+                "length": 6,
+                "requires": [
+                  {"or": [
+                    "heatProof",
+                    "canHeatRun"
+                  ]},
+                  {"cost": [
+                    {"heatFrames": 100}
+                  ]}
+                ]
+              },
               {
                 "length": 12,
-                "requires": null
+                "requires": [
+                  {"or": [
+                    "heatProof",
+                    "canHeatRun"
+                  ]},
+                  {"or": [
```

Comment:
Would charge plasma be a comparable time?

Reply:
Also, there should probably a way to use this by killing the pirate by any slower means (except maybe morph bombs)? We could have it require heatproof at worst.

Reply:
Adding canKillYellowSpacePirates to the heatProof part. Charge+Plasma became reliable when I closed the gap to the pirates a bit so that they won't shoot their beams. Inserting it with a slight heatframes increase to the bunch.